core-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessState.java

```diff
@@ -227,4 +230,18 @@ public String getWaitToken() {
   public void setWaitToken(@Nullable String waitToken) {
     myWaitToken = waitToken;
   }
+
+  /**
+   * Returns whether the state is set to listen in the background
```

Comment:
Agreed. It might, however,  be useful to elaborate on exactly what listening in the background means.